TeachingRecordSystem/src/TeachingRecordSystem.Core/Dqt/Models/GeneratedOptionSets.cs

```diff
@@ -1208,6 +1208,10 @@ public enum dfeta_classdivision
 			" a non-honours course", 9)]
 		Ordinary_includingdivisionsofordinaryifanydegreeawardedafterfollowinganonhonourscourse = 389040009,
 		
+		[System.Runtime.Serialization.EnumMemberAttribute()]
```

Comment:
We may have to manually remove this enum entry until this is deployed to live.

Reply:
Leaving merge until CRM changes are deployed